sdk/storage/storage-blob/test/node/sas.spec.ts

```diff
@@ -188,7 +188,7 @@ describe("Shared Access Signature (SAS) generation Node.js only", () => {
 
     const containerSAS = generateBlobSASQueryParameters(
       {
-        containerName,
+        containerName: containerClient.containerName,
```

Comment:
Is this necessary, or just to get additional test coverage?  Looks like they should have the same value.

Reply:
Just an additional check